cg/services/delivery_message/messages/rna_delivery_message.py

```diff
@@ -51,31 +51,21 @@ def __init__(self, store, strategy: RNAUploadMessageStrategy):
         self.strategy = strategy
 
     def create_message(self, cases: list[Case]) -> str:
-        if len(cases) == 1:
-            return self._get_case_message(cases[0])
         return self._get_cases_message(cases)
```

Comment:
Can we not move the method body of _get_cases_message into this method now? I don't see the point in splitting them